app/components/task/modal.hbs

```diff
@@ -0,0 +1,23 @@
+{{#if @showModal}}
+    <div class="overlay">
+        <div class="modal">
+            <button
+                type="button"
+                class="close"
+                {{on "click" @goBack}}
+            >
+                <img src="close.png" alt="close" height="20px" width="20px" />
+            </button>
+            <p class="modal-title">{{@message}}</p>
+            {{#if @buttonRequired}}
+                <button
+                    type="button"
+                    class="proceed-btn"
+                    {{on "click" @markComplete}}
+                >
+                    Proceed
+                </button>
+            {{/if}}
+        </div>
+    </div>
+{{/if}}
```

Comment:
Can we please have 2 space indentations instead of 4 space?
We are following 2 indentation space for all the files in website-my